sites/main-site/src/content/blog/2025/modules-ontology.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: "Integrating ontologies into nf-core modules"
+subtitle: "Getting onto the right path"
+headerImage: "https://images.unsplash.com/photo-1530477765758-9fa34af23dad"
+headerImageAlt: "A path made out of wooden planks in the middle of a green forest."
+pubDate: 2025-02-21T14:00:00+01:00
+authors:
+    - "mirpedrol"
+    - "mashehu"
+label:
+    - "tools"
+    - "modules"
+maxHeadingDepth: 3
+---
+
+## Introduction
+
+nf-core modules come with a `meta.yml` file used to describe the structure of that module.
+
+The initial structure of the `meta.yml` file did not represent the structure of the modules input and output channels.
+This structure was describing all channel elements at the top level, without grouping them by channel.
+This led to issues such as missing descriptions for multiple meta maps or difficulties in the identification of different channels.
```

Comment:
```suggestion
Each nf-core module includes a `meta.yml` file that describes its structure. However, the initial design of this file didn't reflect the actual organization of the module's input and output channels. Instead of grouping elements by channel, they were all listed at the top level. This made it difficult to understand the channel structure and led to problems such as missing descriptions for multiple meta maps, and difficulties understanding channel structure.
```